servers/xr/xr_positional_tracker.cpp

```diff
@@ -68,8 +68,8 @@ void XRPositionalTracker::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_input", "name", "value"), &XRPositionalTracker::set_input);
 	ADD_SIGNAL(MethodInfo("button_pressed", PropertyInfo(Variant::STRING, "name")));
 	ADD_SIGNAL(MethodInfo("button_released", PropertyInfo(Variant::STRING, "name")));
-	ADD_SIGNAL(MethodInfo("input_float_changed", PropertyInfo(Variant::STRING, "name"), PropertyInfo(Variant::FLOAT, "value")));
-	ADD_SIGNAL(MethodInfo("input_vector2_changed", PropertyInfo(Variant::STRING, "name"), PropertyInfo(Variant::VECTOR2, "vector")));
+	ADD_SIGNAL(MethodInfo("input_float_changed", PropertyInfo(Variant::STRING, "name"), PropertyInfo(Variant::FLOAT, "value"), PropertyInfo(Variant::FLOAT, "prev_value")));
```

Comment:
Isn't this backwards with what the logic below does? It seems to me like you're passing the old value first, then the new value.

Reply:
old value is second so it doesn't break old bindings to the signal because if you have code which was previously bound it should still work

Reply:
I understand the rationale, but the code is still wrong :)

```
case Variant::FLOAT: {
				emit_signal(SNAME("input_float_changed"), p_action_name, inputs[p_action_name], p_value);
```

This emits `input_float_changed` with `name, prev_value, new_value`, which is not what is expected from the name of the signal parameters.

Reply:
yeah as said below, its a draft PR. I made typos my laptop was building and I hadn't tested yet I just wanted to run the CI because I've been bit by style issues in the past